encrypted-media-respec.html

```diff
@@ -647,31 +552,37 @@ <h2>Obtaining Access to Key Systems</h2>
       <section>
         <h3>Permissions Policy Integration</h3>
         <p>
-          <a def-id="requestMediaKeySystemAccess"></a> is a <a
-          data-cite="!PERMISSIONS-POLICY#policy-controlled-feature">policy-controlled feature</a>
-          identified by the string <dfn><code>encrypted-media</code></dfn>.  Its <a
-          data-cite="!PERMISSIONS-POLICY#default-allowlist">default allowlist</a> is <code>'self'</code>
+          {{Navigator/requestMediaKeySystemAccess()}} is a [=policy-controlled feature=]
+          identified by the string <dfn><code>encrypted-media</code></dfn>.  Its
+          [=policy-controlled feature/default allowlist=] is <code>'self'</code>
           [[!PERMISSIONS-POLICY]].
         </p>
       </section>
 
-      <section>
+      <section data-dfn-for="Navigator">
         <h3><dfn>Navigator</dfn> Extension: <code>requestMediaKeySystemAccess()</code></h3>
 
-        <pre class="idl">[Exposed=Window]
-partial interface Navigator {
-    [SecureContext] Promise&lt;MediaKeySystemAccess&gt; requestMediaKeySystemAccess (DOMString keySystem, sequence&lt;MediaKeySystemConfiguration&gt; supportedConfigurations);
-};</pre><section><h2>Methods</h2><dl class="methods" data-dfn-for="Navigator" data-link-for="Navigator"><dt><dfn><code>requestMediaKeySystemAccess</code></dfn></dt><dd>
+        <pre class="idl">
+          [Exposed=Window]
+          partial interface Navigator {
+            [SecureContext] Promise&lt;MediaKeySystemAccess&gt; requestMediaKeySystemAccess (DOMString keySystem, sequence&lt;MediaKeySystemConfiguration&gt; supportedConfigurations);
+          };
+        </pre>
+        <section>
+          <h2>Methods</h2>
+          <dl class="methods">
+          <dt><dfn><code>requestMediaKeySystemAccess()</code></dfn></dt>
```

Comment:
More FYI than requiring a change, but here and below, there is no need to have `<code>` in these definitions. ReSpec is smart enough to know that this is a definition of an IDL method of `Navigator` and it will wrap the definition in `<code>` itself if it's not there.

Reply:
OK, I'll do this in a subsequent commit.

Reply:
Now done throughout the spec.